### PR TITLE
set sequencer inbox maxtimevariation action contract

### PR DESCRIPTION
**Pull request by DZGoldman**

### CODE REVIEW

File: src/gov-action-contracts/AIPs/SetSeqMaxTimeVariation/AIPSetSequencerInboxMaxTimeVariationArbOneAction.sol [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import "../../sequencer/SetSequencerInboxMaxTimeVariationAction.sol";
+
+contract AIPSetSequencerInboxMaxTimeVariationArbOneAction is
+    SetSequencerInboxMaxTimeVariationAction
+{
+    // TODO: confirm / finalize values
+    constructor()
+        SetSequencerInboxMaxTimeVariationAction(
+            ISequencerInboxGetter(0xd514C2b3aaBDBfa10800B9C96dc1eB25427520A0),
+            5760,
```
Comment (fredlacs) [inline src/gov-action-contracts/AIPs/SetSeqMaxTimeVariation/AIPSetSequencerInboxMaxTimeVariationArbOneAction.sol:14]: Would be good to document what these values mean

File: src/gov-action-contracts/AIPs/SetSeqMaxTimeVariation/AIPSetSequencerInboxMaxTimeVariationArbOneAction.sol [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import "../../address-registries/interfaces.sol";
+import "../../sequencer/SetSequencerInboxMaxTimeVariationAction.sol";
+
+contract AIPSetSequencerInboxMaxTimeVariationArbOneAction is
+    SetSequencerInboxMaxTimeVariationAction
+{
+    // TODO: confirm / finalize values
+    constructor()
+        SetSequencerInboxMaxTimeVariationAction(
+            ISequencerInboxGetter(0xd514C2b3aaBDBfa10800B9C96dc1eB25427520A0),
```
Comment (fredlacs) [inline src/gov-action-contracts/AIPs/SetSeqMaxTimeVariation/AIPSetSequencerInboxMaxTimeVariationArbOneAction.sol:13]: Why don't we use the address registry like in other action contracts?

File: src/gov-action-contracts/AIPs/SetSeqMaxTimeVariation/AIPSetSequencerInboxMaxTimeVariationArbOneAction.sol (hunk 1 above)
Comment (fredlacs) [reply src/gov-action-contracts/AIPs/SetSeqMaxTimeVariation/AIPSetSequencerInboxMaxTimeVariationArbOneAction.sol:14]: Why these values are being set and what they represent

File: src/gov-action-contracts/AIPs/SetSeqMaxTimeVariation/AIPSetSequencerInboxMaxTimeVariationArbOneAction.sol (hunk 2 above)
Comment (DZGoldman) [reply src/gov-action-contracts/AIPs/SetSeqMaxTimeVariation/AIPSetSequencerInboxMaxTimeVariationArbOneAction.sol:13]: this is the address registry — it's instantiated w/ the interface w/ only the getter we need

File: src/gov-action-contracts/AIPs/SetSeqMaxTimeVariation/AIPSetSequencerInboxMaxTimeVariationArbOneAction.sol [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    SetSequencerInboxMaxTimeVariationAction
+{
+    // TODO: confirm / finalize values
+    constructor()
+        SetSequencerInboxMaxTimeVariationAction(
+            ISequencerInboxGetter(0xd514C2b3aaBDBfa10800B9C96dc1eB25427520A0), // Arb One Address Registry
+            5760, // Delay blocks (same as current value)
+            64, // New future blocks value
+            86_400, //  Delay seconds (same as current value)
+            768 // Delay seconds (delay blocks * 12)
```
Comment (PlasmaPower) [inline src/gov-action-contracts/AIPs/SetSeqMaxTimeVariation/AIPSetSequencerInboxMaxTimeVariationArbOneAction.sol:18]: nit: this should be "future seconds" I think

File: src/gov-action-contracts/AIPs/SetSeqMaxTimeVariation/AIPSetSequencerInboxMaxTimeVariationNovaAction.sol [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    SetSequencerInboxMaxTimeVariationAction
+{
+    // TODO: confirm / finalize values
+    constructor()
+        SetSequencerInboxMaxTimeVariationAction(
+            ISequencerInboxGetter(0x2F06643fc2CC18585Ae790b546388F0DE4Ec6635), // Nova Address Registry
+            5760, // Delay blocks (same as current value)
+            64, // New future blocks value
+            86_400, //  Delay seconds (same as current value)
+            768 // Delay seconds (delay blocks * 12)
```
Comment (PlasmaPower) [inline src/gov-action-contracts/AIPs/SetSeqMaxTimeVariation/AIPSetSequencerInboxMaxTimeVariationNovaAction.sol:18]: same thing here

File: src/gov-action-contracts/AIPs/SetSeqMaxTimeVariation/AIPSetSequencerInboxMaxTimeVariationArbOneAction.sol [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+pragma solidity 0.8.16;
+
+import "../../address-registries/interfaces.sol";
+import "../../sequencer/SetSequencerInboxMaxTimeVariationAction.sol";
+
+/// @notice Set the future blocks / future seconds to 64 blocks and 64 * 12 seconds so that small L1 reorgs don't cause batches to revert. Delay blocks / delay seconds remain their current values.contract
+contract AIPSetSequencerInboxMaxTimeVariationArbOneAction is
+    SetSequencerInboxMaxTimeVariationAction
+{
+    // TODO: confirm / finalize values
```
Comment (yahgwai) [inline src/gov-action-contracts/AIPs/SetSeqMaxTimeVariation/AIPSetSequencerInboxMaxTimeVariationArbOneAction.sol:11]: Hanging todo in both

**Review by yahgwai — APPROVED** [review]
I'm not sure how to do it, but it would be nice if we could have some simple unit tests for these actions.

**Comment by DZGoldman** [comment]
@yahgwai added test

**Comment by gzeoneth** [comment]
matches deployment of
SetSequencerInboxMaxTimeVariationArbOneAction https://etherscan.io/address/0x3e313eeed58e851ca3841c6109697b9eb35c7726
SetSequencerInboxMaxTimeVariationNovaAction https://etherscan.io/address/0x47a85c0a118127f3968a6a1a61e2a326517540d4

**Pull request merged**